Publish tf_static more frequently

hect95 opened the pull request:
- Add a timer to publish `tf_static` at the frequency set by the `static_publish_frequency` parameter.
- Improve `tf_static` capture in ROSbags, especially when using the splitting feature.

hect95 committed:
Remove tf_static pub

hect95 committed:
Use static tf broadcaster instead

- Add static_publish_frequency to define publish static tf timer

GreatAlexander commented on src/robot_state_publisher.cpp:
```diff
@@ -127,6 +127,7 @@ RobotStatePublisher::RobotStatePublisher(const rclcpp::NodeOptions & options)
   this->declare_parameter("ignore_timestamp", false);
 
   tf_broadcaster_ = std::make_unique<tf2_ros::TransformBroadcaster>(this);
+  
```
Why was this space line added?

hect95 replied:
Space line removed

hect95 committed:
Publish fixed transforms immediately upon initialisation

- Avoids waiting for the first timer interval.
- Useful for long intervals, e.g., 10 seconds.

hect95 commented:
@GreatAlexander Could you aprove :)?
